browser/net/brave_ad_block_tp_network_delegate_helper.cc

```diff
@@ -169,13 +163,10 @@ int OnBeforeURLRequest_AdBlockTPPreWork(
     return net::OK;
   }
 
-  g_brave_browser_process->ad_block_service()->
-        GetTaskRunner()->PostTaskAndReply(FROM_HERE,
-          base::Bind(&OnBeforeURLRequestAdBlockTPOnTaskRunner, ctx),
-          base::Bind(base::IgnoreResult(
-              &OnBeforeURLRequestDispatchOnIOThread), next_callback, ctx));
+  // TODO(bridiver) - fix this
+  OnBeforeURLRequestAdBlockTP(ctx);
```

Comment:
Can you elaborate on what you want to fix here?

Reply:
actually the comment can just be removed, I had that there as a reminder from before our conversation